app/scripts/configs/tracks-info.js

```diff
@@ -682,7 +706,7 @@ export const TRACKS_INFO = [
     aliases: ['horizontal-1d-value-interval', 'vertical-1d-value-interval'],
     datatype: ['bed-value'],
     local: false,
-    orientation: ['1d-horizontal'],
+    orientation: '1d-horizontal',
```

Comment:
Think this was a bug. The track renderer assumed `orientation` was a `string` for all data types. @pkerpedjiev, could `orientation` be an `Array<string>`?